requirements-dev-airflow.txt

```diff
@@ -0,0 +1 @@
+apache-airflow[s3]>=1.9.0
```

Comment:
I think we should get rid of the airflow file... I only added it because there was an extras_require entry in setup.py, but it's not documented and nobody would miss it.